test/functional/wallet_blank.py

```diff
@@ -27,11 +27,23 @@ def skip_test_if_missing_module(self):
     def add_options(self, options):
         self.add_wallet_options(options)
 
+    def test_createwallet(self):
```

Comment:
In commit "rpc, wallet: Set blank when disable private keys in createwallet" (37dd054d9c7d58e829338c88df836241c0259f99)

These tests aren't actually being invoked in the current version of the PR. Need to call `test_createwallet` and `test_createwallet_override` functions in `run_test` below

Reply:
Removed this test so this is no longer relevant.